cerberus-cli/src/main/java/ca/on/oicr/gsi/cerberus/Cerberus.java

```diff
@@ -135,6 +134,7 @@ public static void main(String[] args) {
 
         ProvenanceHttpClient client = new ProvenanceHttpClient(uri);
 
+        // TODO validate inputs, eg. type and action are in respective enums
```

Comment:
Rather than a code comment, can this be placed in a JIRA ticket?

Reply:
In fact that comment is now obsolete. Type and action are checked on lines 99-117 of Cerberus.java. I'll remove the comment.